src/NativeShell/www/nativeshell.js

```diff
@@ -0,0 +1,48 @@
+
+
+function invokeMethod(successCallback, errorCallback, method, options) {
+    successCallback = successCallback || function () {};
+    errorCallback = errorCallback || function () {};
+    options = options || [];
+
+    cordova.exec(successCallback, errorCallback, 'NativeShell', method, options);
+}
+
+function NativeShell() {
+    this.FileSystem = require('org.jellyfin.mobile.FileSystem');
+    this.AppHost = require('org.jellyfin.mobile.AppHost');
+}
+
+NativeShell.prototype.getDeviceInformation = function(successCallback, errorCallback) {
+    invokeMethod(successCallback, errorCallback, 'getDeviceInformation', []);
+};
+
+NativeShell.prototype.enableFullscreen = function(successCallback, errorCallback) {
+    invokeMethod(successCallback, errorCallback, 'enableFullscreen', []);
+};
+
+NativeShell.prototype.disableFullscreen = function(successCallback, errorCallback) {
+    invokeMethod(successCallback, errorCallback, 'disableFullscreen', []);
+};
+
+NativeShell.prototype.updateMediaSession = function(options, successCallback, errorCallback) {
+    invokeMethod(successCallback, errorCallback, 'updateMediaSession', [options]);
+};
+
+NativeShell.prototype.hideMediaSession = function(successCallback, errorCallback) {
+    invokeMethod(successCallback, errorCallback, 'hideMediaSession', []);
+};
+
+NativeShell.prototype.openUrl = function (url, target) {
+    cordova.InAppBrowser.open(url, target || '_system');
+};
+
+NativeShell.prototype.sync = function(successCallback, errorCallback) {
```

Comment:
I think this can be dropped

Reply:
The code related to this on jf-web should also be removed